src/components/ThemeContainer/ThemeContainer.js

```diff
@@ -0,0 +1,38 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getThemeName } from 'selectors/theme';
+import { connect } from 'react-redux';
+
+export class ThemeContainer extends Component {
+  static propTypes = {
+    theme: PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    document.body.classList.add('theme--default');
+  }
+
+  componentDidUpdate() {
+    if (this.props.theme === 'default') {
+      document.body.classList.add('theme--default');
+      document.body.classList.remove('theme--dark');
+    } else {
+      document.body.classList.add('theme--dark');
+      document.body.classList.remove('theme--default');
+    }
+  }
+
+  render() {
+    return null;
```

Comment:
Any time a component returns null, it's a good sign that it should be a [higher-order component](https://reactjs.org/docs/higher-order-components.html) instead - it doesn't need to exist in the DOM tree.